core/src/services/redis/fixtures/redis_cluster/docker-compose.yml

```diff
@@ -0,0 +1,92 @@
+version: '3.8'
+
+services:
+  redis-node-master-1:
+    image: redis:latest
```

Comment:
How about using https://github.com/bitnami/containers/blob/main/bitnami/redis-cluster/README.md instead? I prefer to configure container via config file and env instead of command.